app/services/intercom_service.rb

```diff
@@ -129,11 +129,22 @@ def self.intercom
     @intercom ||= Intercom::Client.new(token: EnvironmentCredentials.dig(:intercom, :intercom_access_token))
   end
 
+  MAX_RETRY_COUNT = 3
+
   def self.intercom_api(collection, verb, params)
     if Rails.env.development?
       Rails.logger.debug("Calling Intercom: #{collection}.#{verb}(#{params})")
     end
-    result = intercom.send(collection).send(verb, params)
+
+    retry_counts = 0
+
+    begin
+      result = intercom.send(collection).send(verb, params)
+    rescue
+      retry_counts += 1
+      retry if retry_counts < MAX_RETRY_COUNT
```

Comment:
could consider booping a Datadog counter whenever this retries, like in https://github.com/codeforamerica/vita-min/blob/main/app/services/efile/poll_for_acknowledgments_service.rb#L16

that way we could tell how often we're getting into this rescue at all